pipeline/outputs/s3.md

```diff
@@ -25,6 +25,7 @@ Records are stored in files in S3 as newline delimited JSON.
 | upload\_chunk\_size | The size of each 'part' for multipart uploads. Max: 50M | 5,242,880 bytes |
 | upload\_timeout | Whenever this amount of time has elapsed, Fluent Bit will complete an upload and create a new file in S3. For example, set this value to 60m and you will get a new file every hour. | 10m |
 | store\_dir | Directory to locally buffer data before sending. When multipart uploads are used, data will only be buffered until the `upload_chunk_size` is reached. | /tmp/fluent-bit/s3 |
+| log\_key | By default, the whole log record will be sent to S3. If you specify a key name with this option, then only the value of that key will be sent to S3. For example, if you are using the Fluentd Docker log driver, you can specify log_key log and only the log message will be sent to S3. | None |
```

Comment:
nit: replace "if you are using the Fluentd Docker log driver" with "if you are using Docker"